docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+
+The last mile problem is actually a great opportunity for RAG apps. Despite the eagerness of major AI actors to become encompassing platforms, the last mile won’t be easily solved. The right fit for a use case, a corpus, a specific user target is the hardest part of a RAG system, probably the only actual moat there is in applied generative AI.
+
+In turn, this means that the biggest issue for RAG is evaluation. Preliminary evaluations and benchmarks to select and swap models is not the hardest part — you could even do with synthetic data. The real challenge is evaluation in production of the different components of your system, taking into account their level of specialization. While we strive right now to automate everything, evaluation is a final residual of qualitative human assessment.
+
+## 2. Retrieval: back to future.
+As RAG was primarily formulated in an LLM/embedding context, it obfuscated the actual roots of the communication system it came to develop: information retrieval.
+
+### 2.1 The actual roots of RAG: decades of IR and data frictions.
+Expert systems and knowledge infrastructures have been existing for decades and dealt with the same data frictions that characterize RAG applications today. Frictions at the entry level, with much additional work needed to fit new content to imperative formats and expectations. Frictions on the receiving ends, as users are not behaving according to the expectations of the search infrastructure.
+
+Back in the 1960s, the first expert systems widely underestimated the massive intermediation work needed to get these systems working. It was hoped that they would soon be seamlessly accessed by professionals: “MEDLINE was intended to be used by medical researchers and clinicians, NASA/RECON was designed for aerospace engineers and scientists.” And yet, “For many reasons, however, most users through the seventies were librarians and trained intermediaries working on behalf of end users.”
+
+It sounds like a familiar pattern. RAG emerged with a similar promise of simplification shock: taking whatever content there is, outputting whatever content the user wants. This is not only the obvious objective of LLM on the output side but also embedding models on the input side. Embeddings are natively multilingual, theoretically resilient to synonyms, variations of language standards and document artifacts. 
+
+The radical promise frequently failed on two fronts: understanding users, understanding data.
+
+### 2.2 Know your users
+This is the obvious start of most Information retrieval projects but, until now, I haven’t seen it frequently mentioned in a RAG context: know what your users are doing with your systems.
+
```

Comment:
"until now" = until you mention it now? until recently?

Reply:
until recently but can be dropped.